e2e/test/scenarios/sharing/downloads/downloads.cy.spec.js

```diff
@@ -65,21 +65,30 @@ describe("scenarios > question > download", () => {
     cy.signInAsAdmin();
   });
 
-  testCases.forEach(fileType => {
-    it(`downloads ${fileType} file`, () => {
-      startNewQuestion();
-      entityPickerModal().within(() => {
-        entityPickerModalTab("Saved questions").click();
-        cy.findByText("Orders, Count").click();
-      });
+  describeWithSnowplow("[with snowplow events]", () => {
```

Comment:
I just wrapped this with describeWithSnowplow and added the expect `expectGoodSnowplowEvent` call